src/plugins/workspace/public/components/workspace_name/workspace_name_with_icon.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiIcon } from '@elastic/eui';
+import { WorkspaceAttribute } from 'opensearch-dashboards/public';
+import React from 'react';
+import { getUseCase } from '../../utils';
+import { WorkspaceUseCase } from '../../types';
+
+interface Props {
+  workspace: WorkspaceAttribute;
+  availableUseCases: WorkspaceUseCase[];
+}
+
+export const WorkspaceNameWithIcon = ({ workspace, availableUseCases }: Props) => {
```

Comment:
How about call it `WorkspaceTitleDisplay`?

Reply:
make sense, will update it accordingly.